lua/unicode.lua

```diff
@@ -0,0 +1,30 @@
+-- Unicode
+-- 复制自： https://github.com/shewer/librime-lua-script/blob/main/lua/component/unicode.lua
+-- 示例：输入 U62fc 得到「拼」
+-- 触发前缀默认为 recognizer/patterns/unicode 的第 2 个字符，即 U
+-- 2024.02.26: 限定编码最大值
+local function unicode(input, seg, env)
+    -- 获取 recognizer/patterns/unicode 的第 2 个字符作为触发前缀
+    env.unicode_keyword = env.unicode_keyword or
+        env.engine.schema.config:get_string('recognizer/patterns/unicode'):sub(2, 2)
```

Comment:
应该防御没有设置 recognizer pattern 的错误情况

```suggestion
        env.engine.schema.config:get_string('recognizer/patterns/unicode'):sub(2, 2) or 'U'
```